<!-- -*- mode: XML -*- -->
<launch>

    <arg name="SaveFolder" default="/home/pro/data/temp/"/>
    <arg name="isShow" default="false"/>
    <arg name="isSave" default="false"/>
    <arg name="serial_port" default="/dev/ttyVideo0"/>    

    <node pkg="frame" type="frame" name="frame" output="screen">
        <param name="SaveFolder" value="$(arg SaveFolder)"/>
        <param name="isShow" value="$(arg isShow)"/>
        <param name="isSave" value="$(arg isSave)"/>
        <param name="serial_port" value="$(arg serial_port)"/>               
    </node>

</launch>
